backend/pokebattle/settings/local.py

```diff
@@ -1 +1,9 @@
 from .local_base import *  # noqa
+from decouple import config  # noqa
```

Comment:
`local.py` should be out of the version control system. `base_local.py` is where you set de  defaults for local development and `local.py` is for personal configs that don't go into VCS